### PR TITLE
Bm improve test

### PR DESCRIPTION
**Pull request by bolom**

### CODE REVIEW

File: lib/safe_eth_ruby/protocol.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 require "abi_coder_rb"
-module Safe
+
+module SafeEthRuby
   class Protocol
     attr_reader :safe_api, :signer, :chain_id, :safe_address
 
     def initialize(signer:, chain_id:, safe_address:, rpc:)
+      puts SafeEthRuby.inspect
```
Comment (vokshirg) [inline lib/safe_eth_ruby/protocol.rb:10]: forgot debug

File: test/test_helper.rb [hunk 2]
```diff
@@ -7,5 +7,5 @@
 require "minitest/autorun"
 
 $LOAD_PATH.unshift(File.join(File.expand_path(__dir__), "..", "lib"))
-require "safe.rb"
+require "safe_eth_ruby.rb"
```
Comment (vokshirg) [inline test/test_helper.rb:10]: The Ruby interpreter will automatically look for a .rb file when requiring dependencies.

**Review by vokshirg — COMMENTED** [review]
small comments

**Comment by bolom** [comment]
@vokshirg thanks a lot for your feedback

**Pull request merged**